<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingBehaviorForm</class>
 <widget class="QWidget" name="SettingBehaviorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Behavior</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_auto_fetch">
     <property name="text">
      <string>Automatically fetch when opening the repository</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_avatar_icon_provider">
     <property name="title">
      <string>Use avatar icon provider</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_avatar_provider_gravatar">
        <property name="text">
         <string>gravatar.com</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_avatar_provider_libravatar">
        <property name="text">
         <string>libravatar.org</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Maximum number of commit item acquisitions</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_max_commit_item_acquisitions">
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>GPG signing policy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_signing_policy">
       <property name="text">
        <string>Configure...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>275</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBox_auto_fetch</tabstop>
  <tabstop>spinBox_max_commit_item_acquisitions</tabstop>
  <tabstop>pushButton_signing_policy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
